src/pages/Templates.svelte

```diff
@@ -66,6 +68,8 @@
             window.location.reload();
         }
     }
+
+    $: previewFontSize = templateWidth / 21.8;
```

Comment:
What is the 21.8 magic number?  Can this be a const listed somewhere above with a name for the logic of how it's constructed?  Like STANDARD_TEMPLATE_WIDTH = 21.8 or something.  I try to avoid magic numbers as much as possible and instead opt for a constant with comments elsewhere (otherwise, I forget why the magic number is needed).

Reply:
Currently it's an experimentally determined value, but I will make this more accurate. It will be the ratio of the template width / font size from server/dal/cardApi.js. It is used with the width of the whole template element to determine what the preview text font size should be. I'll make this a const and add a comment.